src/main/java/com/solvd/pages/common/DrawingPageBase.java

```diff
@@ -0,0 +1,13 @@
+package com.solvd.pages.common;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class DrawingPageBase extends PageBase {
+    public DrawingPageBase(WebDriver driver) {
```

Comment:
it's a good practice to leave a blank line between the class declaration and the first field